src/main/kotlin/de/gesellix/docker/compose/ComposeFileReader.kt

```diff
@@ -2,17 +2,7 @@ package de.gesellix.docker.compose
 
 import com.squareup.moshi.Moshi
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
-import de.gesellix.docker.compose.adapters.ListToExposeAdapter
-import de.gesellix.docker.compose.adapters.ListToPortConfigsAdapter
-import de.gesellix.docker.compose.adapters.ListToServiceSecretsAdapter
-import de.gesellix.docker.compose.adapters.ListToServiceVolumesAdapter
-import de.gesellix.docker.compose.adapters.MapOrListToEnvironmentAdapter
-import de.gesellix.docker.compose.adapters.MapOrListToExtraHosts
-import de.gesellix.docker.compose.adapters.MapOrListToLabelAdapter
-import de.gesellix.docker.compose.adapters.MapToDriverOptsAdapter
-import de.gesellix.docker.compose.adapters.MapToExternalAdapter
-import de.gesellix.docker.compose.adapters.StringOrListToCommandAdapter
-import de.gesellix.docker.compose.adapters.StringToServiceNetworksAdapter
+import de.gesellix.docker.compose.adapters.*
```

Comment:
Can you change the wildcard imports back to explicit ones? Same applies to the other changed files in this pull request, too.